### ISSUE TITLE
Applying positionType absolute the display does not work

### ISSUE THREAD

**Issue by luisgustavozup** [ZupIT/beagle#1421]
## Description

When you apply the absolute position Type attribute you cannot define the NONE display.
Soon the view is not invisible.

## Steps To Reproduce

Link Playground: https://playground.usebeagle.io/#/cloud/ea4f3a1bd3e84c2b9fa6bf06a97ddaa7/BugDisplay?platform=react-web

```
{
  "_beagleComponent_": "beagle:screenComponent",
  "child": {
    "_beagleComponent_": "beagle:container",
    "children": [
      {
        "_beagleComponent_": "beagle:text",
        "text": "1"
      },
      {
        "_beagleComponent_": "beagle:container",
        "children": [
          {
            "_beagleComponent_": "beagle:text",
            "text": "2"
          }
        ],
        "style": {
          "backgroundColor": "#EBF04E",
          "size": {
            "width": {
              "value": 100,
              "type": "PERCENT"
            },
            "height": {
              "value": 100,
              "type": "PERCENT"
            }
          },
          "positionType": "ABSOLUTE",
          "display": "NONE"
        }
      }
    ],
    "style": {
      "backgroundColor": "#82E09A",
      "flex": {
        "grow": 1
      }
    }
  }
}
```

<img width="351" alt="Screen Shot 2021-03-09 at 14 10 00" src="https://user-images.githubusercontent.com/59032921/110509719-2d3e8100-80e1-11eb-9ba7-8612ba248667.png">

## Expected Results
<img width="577" alt="Screen Shot 2021-03-09 at 14 08 38" src="https://user-images.githubusercontent.com/59032921/110509552-fbc5b580-80e0-11eb-9faa-5ead64948299.png">

**Comment by luisgustavozup** [comment]
The yoga layout repository fixed this problem.
Link: https://github.com/facebook/yoga/commit/342aebe1d73e5770a1862b6a94c6b877c1439a9b

**Comment by Tiagoperes** [comment]
Issue #1437 might solve this

**Comment by dantes-git** [comment]
Solved with #1511
